hirola/front/templates/front/before_checkout.html

```diff
@@ -4,45 +4,48 @@
 {% endblock %} {% block content %}
 {% load before_cart %}
 {% load humanize %}
+{% load stars_counter %}
 <div class="container main-content">
     <div class="row cart-content">
         {% if items %}
         {% for item in items %}
-        <div class="col m8 s12 right-content">
-                <div class="row heading-right"><h5>Teke sale</h5></div>
+        <div class="col s12 m8 l8 right-content">
+                <div class="row heading-right"><h5>{teke}<sup>2</sup> hot deal</h5></div>
                 <div class="row">
-                    <div class="col m3">
+                    <div class="col s6m4">
```

Comment:
put a space between the two materialize classes for them to be valid

Reply:
@dragz92 work on this on the next P.R. I need this one merged.